src/omv/modules/py_image.c

```diff
@@ -578,20 +561,22 @@ static mp_obj_t py_image_subscr(mp_obj_t self_in, mp_obj_t index, mp_obj_t value
                         mp_obj_get_array(value, &value_l_len, &value_l);
                         PY_ASSERT_TRUE_MSG(value_l_len == (slice.stop - slice.start), "cannot grow or shrink image");
                         for (mp_uint_t i = 0; i < (slice.stop - slice.start); i++) {
-                            self->_cobj.data[slice.start + i] = mp_obj_get_int(value_l[i]);
+                            image->data[slice.start + i] = mp_obj_get_int(value_l[i]);
                         }
                     } else {
                         mp_int_t v = mp_obj_get_int(value);
                         for (mp_uint_t i = 0; i < (slice.stop - slice.start); i++) {
-                            self->_cobj.data[slice.start + i] = v;
+                            image->data[slice.start + i] = v;
                         }
                     }
                     return mp_const_none;
                 }
-                mp_uint_t i = mp_get_index(self->base.type, self->_cobj.bpp, index, false);
-                self->_cobj.data[i] = mp_obj_get_int(value);
+                mp_uint_t i = mp_get_index(self->base.type, image->bpp, index, false);
+                image->data[i] = mp_obj_get_int(value);
                 return mp_const_none;
             }
+            default:
```

Comment:
Braces

Reply:
Not needed unless you declare variables.